src/main/java/org/fogbowcloud/sebal/SEBALHelper.java

```diff
@@ -115,14 +115,14 @@ public static int centralMeridian(int zoneNumber) throws Exception {
 			StringTokenizer tokenizedResponse = new StringTokenizer(responseStr, "\n");
```

Comment:
remove above string from the code path http://www.spatialreference.org/ref/epsg/327" + zoneNumber + "/prettywkt/")

create a constant to it (it would be event better to have it in a config flie)